src/modules/75-cd/components/Environments/EnvironmentList/EnvironmentsList.tsx

```diff
@@ -7,7 +7,7 @@
 
 import React, { useEffect, useMemo, useState } from 'react'
 import cx from 'classnames'
-import { Pagination, Layout, Text, Container, Heading, TableV2 } from '@wings-software/uicore'
+import { Pagination, Layout, Text, Container, Heading, TableV2, ButtonVariation } from '@wings-software/uicore'
```

Comment:
if you're pushing changes anyway :) 

```suggestion
import { Pagination, Layout, Text, Container, Heading, TableV2, ButtonVariation } from '@harness/uicore'
```

Reply:
Unfortunately can't fix this as its not our module, have removed the change altogether